### ISSUE TITLE
clarification of contribution and citation format

### ISSUE THREAD

**Issue by snakehaihai** [yutongwangBIT/VOOM#1]
Hi, first of all, your Bibtex reference on this GitHub repo seems to have an issue.

Second, how it is different from a TRO paper called "An Object SLAM Framework for Association, Mapping, and High-Level Tasks." that appeared a few month before the ICRA submission.  The concept and drawing look very familiar with that. 
See figure below 

![image](https://github.com/yutongwangBIT/VOOM/assets/6070995/797a0118-f15b-4af2-8581-fbf40fd59559)

**Comment by brytsknguyen** [comment]
TRO paper not open sourced...

**Comment by snakehaihai** [comment]
> TRO paper not open sourced...

TRO paper based on this
https://arxiv.org/abs/2004.12730

https://github.com/yanmin-wu/EAO-SLAM?tab=readme-ov-file

**Comment by snakehaihai** [comment]
checked. besides, it didn't compare with EAO. no other major issue. backend code and method looks different

**Comment by yutongwangBIT** [comment]
Hi Dr. Yuan,

Thank you so much for diving into our work and sharing your thoughts. 

Here's what we've got after reading your comments:

1. **On the distinction between our work and others**: Our research and the studies you mentioned like EAO-SLAM and the TRO paper all build upon ORB-SLAM2, leveraging both feature points and dual quadrics as landmarks. However, our focus is distinct. The mentioned studies primarily optimize object poses and introduce new data association methods based on the two types of landmarks. In contrast, our work's main contribution lies in leveraging object information to enhance the localization accuracy of traditional SLAM systems that rely only on feature points. This is a pivotal aspect where our research diverges - our experiments are designed to underscore the improvements in localization accuracy in comparison to other methods, demonstrating the unique value and application of object information within the SLAM domain.
2. **Why we do not compare our results with EAO-SLAM**:Indeed, as you pointed out, our study does not feature a direct comparison with EAO-SLAM, a decision shaped by two significant factors. Initially, attempts to run EAO-SLAM on our setup encountered Segmentation faults, effectively blocking us from gathering comparative results. Moreover, the EAO-SLAM study itself stated that their approach did not boost ORB-SLAM2's localization accuracy, thus deeming a comparative analysis unnecessary. 
3. **Citation update**: By the way, we've updated our manuscript to incorporate the citation of the TRO paper, having previously cited only EAO-SLAM.

We really appreciate your feedback and welcome further discussion. 

Best,
Yutong
